notebooks/user_guide.ipynb

```diff
@@ -2386,6 +2386,18 @@
     "\n",
     "$$\n",
     "        v_{mean} = \\frac{1}{N} \\sum_{i \\in P_M} v_i\n",
+    "$$\n",
+    "\n",
+    "**Gauss speed profiles**\n",
+    "\n",
+    "Calculates a speed profile based on Gaussian weights for an array of pedestrian locations and velocities. The speed, weighted at a grid cell $c$ considering its distance $\\delta = \\boldsymbol{r}_i - \\boldsymbol{c}$ from an agent, is determined as follows:\n",
+    "    \n",
+    "$$     \n",
+    "        v_{c} = \\frac{\\sum_{i=1}^{N}{\\big(w_i(\\delta)\\cdot v_i\\big)}}{\\sum_{i=1}^{N} w_i},\n",
+    "$$\n",
+    "with \n",
+    "$$\n",
+    "        w_i = \\frac{1} {\\sigma \\cdot \\sqrt{2\\pi}} \\exp\\big(-\\frac{\\delta^2}{2\\sigma^2}\\big)\\; \\text{and}\\; \\sigma = \\frac{FWHM}{2\\sqrt{2\\ln(2)}}.\n",
```

Comment:
There seem to be some extra `$` around the formula on the webpage:
![image](https://github.com/PedestrianDynamics/PedPy/assets/15690524/56805481-4253-4ce9-bb32-90c9701e370c)